src/status_im/browser/core.cljs

```diff
@@ -150,21 +155,29 @@
                              :history new-history
                              :history-index new-index)))))
 
+(defmulti storage-gateway :namespace)
+
+(defmethod storage-gateway :ipfs
+  [{:keys [hash]}]
+  (let [base32hash (-> (.encode js-dependencies/hi-base32 (alphabase.base58/decode hash))
+                       (string/replace #"=" "")
+                       (string/lower-case))]
+    (str "https://" base32hash ".infura.status.im")))
```

Comment:
what is this gateway should it be used for all content? there is already a gateway defined in ipfs.core

Reply:
It's necessary for websites. It introduces subdomains to ensure no cookies/localStorage are shared.